src/json/builder.cr

```diff
@@ -280,6 +280,13 @@ class JSON::Builder
     end
   end
 
+  # Returns `true` if the next thing that must pushed into this
```

Comment:
Actually it returns a string (`state.name`) and `false` otherwise.

Reply:
Thanks! But no, it returns a `Bool` (I added a return type just in case).